src/com/ableton/VirtualEnv.groovy

```diff
@@ -149,20 +149,15 @@ class VirtualEnv implements Serializable {
    */
   @SuppressWarnings('MethodReturnTypeRequired')
   def run(Map arguments) {
-    assert arguments
-    assert arguments.containsKey('script')
```

Comment:
Why not assert anymore?

Reply:
My original reasoning was that it wasn't necessary because we were just passing the arguments map along to `sh`, but since you convinced me to re-add the custom label, I guess we should probably return it. :smile: Fixed in 95fc247.